receiver/oracledbreceiver/scraper.go

```diff
@@ -828,3 +828,43 @@ func (s *oracleScraper) shutdown(_ context.Context) error {
 	}
 	return s.db.Close()
 }
+
+func (s *oracleScraper) setupResourceBuilder(rb *metadata.ResourceBuilder) *metadata.ResourceBuilder {
+	rb.SetOracledbInstanceName(s.instanceName)
+	rb.SetHostName(s.hostName)
+	rb.SetServiceInstanceID(s.serviceInstanceID)
+	return rb
+}
+
+func getInstanceID(instanceString string, logger *zap.Logger) string {
+	parts := strings.SplitN(instanceString, "/", 2)
```

Comment:
Is there any valid `instanceString` that has more than one `/`? I'm wondering if there's some edge case we might be missing here.

Reply:
I have the same concern here. Is that possible to use a oracle database library to parse this connection string?

For instance, I use mssql library to parse the string in sql server receiver. https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/a9b21438225993b107ef7efc1eaefbf21547f5a7/receiver/sqlserverreceiver/service_instance_id.go#L76